commands/_update_dabs.js

```diff
@@ -8,13 +9,19 @@ module.exports = function update_dabs(member, config) {
             daily_rolls: 0,
             daily_claim: -1,
         }
-        // TOCONSIDER : Announce that user has free dabs
+        // TOCONSIDER : Announce free dabs
     }
     if (user.daily_claim != new Date().getDay()) {
         user.daily_claim = new Date().getDay()
         user.daily_rolls += Math.floor(10 * (Math.log(user.level) / Math.log(5))) + 1
         // TOCONSIDER : Announce that user has drools
     }
+
+    if (amount === undefined) {
```

Comment:
What? WHY, just do `if (amount !== undefined) {`.
Also the fact that you can do this to dabs means we should just use update_dabs instead of pay_dabs / get_dabs.
Include the error checking for trying to go into minus dabs here.